queens-psyc-clinic/app
# FE Item Card Component

### Card Component
This is the item card component done! Firstly, note I accidentally coded on the `navbar` branch, but its okay because that was already pushed to main, so just ignore the name of the branch please 🙈.

Also, note I didn't add the restriction that it can only be seen on the cilent side as can implement that later as defined in other PRs.  

### Testing
1. navigate to branch: `git checkout navbar`
2. cd to the public folder
3. run `npm install`
4. run `npm start`
5. ensure the cards are showing, no errors, and feel free to checkout the mock data!
<br>

The cards should look like this (there should be a few more hidden when you scroll):

<img width="700" alt="Screenshot 2024-02-26 at 1 05 46 PM" src="https://github.com/queens-psyc-clinic/app/assets/74020381/35ddb87a-5911-47f9-84ee-1ed411f046e2">

For reference see #8
